moto/codebuild/responses.py

```diff
@@ -0,0 +1,222 @@
+from moto.core.responses import BaseResponse
+from .models import codebuild_backends
+from .exceptions import (
+    InvalidInputException,
+    ResourceAlreadyExistsException,
+    ResourceNotFoundException,
+)
+from moto.core import get_account_id
+import json
+import re
+
+
+def _validate_required_params_source(source):
+    try:
+        assert source["type"] in [
```

Comment:
Let's use regular checks here, to keep in line with the code style everywhere else:
```
if x not in y:
    raise Exception(..)
```
There's a few others, I won't point hem all out, but they should all follow the same pattern